vissl/hooks/log_hooks.py

```diff
@@ -201,14 +201,11 @@ def on_update(self, task: "tasks.ClassyTask") -> None:
                         datetime.timedelta(seconds=int(rolling_eta_secs))
                     )
                     rolling_btime = int(1000.0 * rolling_avg_time)
-                    log_data[f"btime({self.btime_freq}iters) (ms)"] = rolling_btime
-                    log_data["rolling_eta"] = rolling_eta_str
-                logging.info(json.dumps(log_data))
-                with PathManager.open(
-                    f"{task.checkpoint_folder}/stdout.json", "a"
-                ) as fopen:
-                    fopen.write(json.dumps(log_data) + "\n")
-                    fopen.flush()
+                    evt_stg.put_scalars(
+                        rolling_btime=rolling_btime, rolling_eta=rolling_eta_str
```

Comment:
The `HistoryBuffer` can only store scalars, so we should only attempt to store a scalar
```suggestion
                        rolling_btime=rolling_btime, rolling_eta=rolling_eta_secs
```

Reply:
yep! only attempt to store scalars.